src/ethereum/trace.py

```diff
@@ -212,6 +216,7 @@ def __call__(
         [evm]: ref:ethereum.frontier.vm.Evm
         [`TraceEvent`]: ref:ethereum.trace.TraceEvent
         """
+        del evm, event
```

Comment:
Does `raise NotImplementedError` also mute the warning? I think raising that error would be more semantically correct.